### ISSUE TITLE
Login failed! Blocked by Google?

### ISSUE THREAD

**Issue by creuzerm** [eyecatchup/GWT_CrawlErrors-php#2]
When I try to use this tool with one of the examples with my credentials, I get a error that simply says "Login failed!"

PHP 5.3.3 on CentOS in with basedir restrictions on.

Doing some debugging, I change GwtCrawlErrors.class.php line 142 to:
            throw new Exception('Login failed! ' . $output);
I see
            Login failed! Error=BadAuthentication 

Knowing that curl and basedir_restrictions don't get along, I comment out line 132
            #CURLOPT_FOLLOWLOCATION => 1,
and try again and get:
            Login failed! Error=BadAuthentication Url=https://www.google.com/accounts/ContinueSignIn?plt=[snip]&sarp=1&scc=1 Info=WebLoginRequired 

I copy and paste the really long url (which I truncated as I think it would allow a sign-in session highjack) into my web-browser and am asked to verify my login by Google. 

I end up with a line in my Google Accounts dashboard, Security, Recent Activity that says that my login has been blocked. I've clicked the "That's Me" button, but it doesn't seem to help, repeating the steps above still gives me a Login Failed! error.

Is this an issue that can be reproduced by anybody else? Any suggestions for getting around it?

**Comment by eyecatchup** [comment]
Thanks for the report. Will have a look at it and report back.

**Comment by eyecatchup** [comment]
@creuzerm What is your `safe_mode` setting? (Not only `open_basedir` but also `safe_mode` can cause issues with `CURLOPT_FOLLOWLOCATION` - it must be set off.)

**Comment by eyecatchup** [comment]
Hmm. It's not the `safe_mode`. I can not reproduce the issue. Neither on my linux server nor on my local Windows machine (see attached screenshot). Before I'm gonna dig deeper into it.. 

Maybe it's just that you login to Google services too often in an automated way (cause the same message was shown to me when I used [my SEOstats class](https://github.com/eyecatchup/SEOstats/) too extensive)? 

Could you verifiy the behaviour from the same machine using a fresh IP and a different account? 

## 

![screenshot_2013_09_17_00_33_15](https://f.cloud.github.com/assets/845131/1153384/738b424a-1f20-11e3-9330-810768c3d764.png)

**Comment by eyecatchup** [comment]
Closed due to invalid/no feedback. Open a new issue if required.

**Comment by fkeeney** [comment]
I'm getting the same error: "Login failed!Error=BadAuthentication"

I've clicked the "That's Me" button at Google but it doesn't seem to help

**Comment by fkeeney** [comment]
Login failed!Error=BadAuthentication Url=https://www.google.com/accounts/ContinueSignIn?sarp= Info=WebLoginRequired

**Comment by fkeeney** [comment]
It worked after I followed this link:
https://accounts.google.com/DisplayUnlockCaptcha
